version/feature_manager.go

```diff
@@ -28,6 +28,7 @@ const (
 	MultiHopSessionFeature
 	IncludeStatusInCli
 	CredentialLibraryVaultSubtype
+	SshIntoTcpTargetAddress
```

Comment:
I find this feature flag name confusing. At first glance it makes it seem like support for sshing into tcp targets with address will be dropped. I think this should be something more like `UseTargetIdForHostId`.

Reply:
👍